tests/_support/Helper/Unit.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Helper;
+
+// here you can define custom actions
+// all public methods declared in helper class will be available in $I
+
+class Unit extends \Codeception\Module {
```

Comment:
Awesome job on this! 

I'm not familiar with Code epitome at all yet. Can you tell me if these empty classes are required?

Reply:
Yes, the files in the _support directory are required https://codeception.com/docs/06-ModulesAndHelpers under `Helpers`

Reply:
The files are actually generated from the bootstrap method https://codeception.com/quickstart